core/arch/arm/crypto/aes-gcm-ce.c

```diff
@@ -1,8 +1,6 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
ugly cpp `//` comment!

Reply:
That's the choice made by the linux kernel guys (well... by Linus himself [1]). Here I'm following the meta-rule "follow the rules of the linux kernel unless we have a valid reason to do otherwise". It will be less problems in the future.

[1] https://patchwork.kernel.org/patch/10062523/ (search for 'happily').

Reply:
ok. if on purpose, i'm fine.